source/Timer.h

```diff
@@ -1,285 +1,72 @@
 /*
```

Comment:
We use lgpl license in rialto-gstreamer not apache

Reply:
changed